qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/pairing_table_widget/pairing_table_widget_presenter.py

```diff
@@ -13,22 +13,16 @@
 from mantidqtinterfaces.Muon.GUI.Common.grouping_tab_widget.grouping_tab_widget_model import RowValid
 from mantidqtinterfaces.Muon.GUI.Common.grouping_table_widget.grouping_table_widget_presenter import row_colors, row_tooltips
 
-
-pair_columns = ["pair_name", "to_analyse", "group_1", "group_2", "alpha"]
+from mantidqtinterfaces.Muon.GUI.Common.pairing_table_widget.pairing_table_constants import get_pair_columns, pair_columns
 
 
 class PairingTablePresenter(object):
     def __init__(self, view, model):
```

Comment:
On the constructor for this class, I think these two slots are not used, could you remove them? 
```
        self._on_alpha_clicked = lambda: 0
        self._on_guess_alpha_requested = lambda pair_name, group1, group2: 0
```

Reply:
I have removed the code from the constructor